app/src/main/res/layout-land/option_app_language.xml

```diff
@@ -23,27 +23,25 @@
 
     <TextView
       android:id="@+id/app_language_label_text_view"
+      style="@style/Subtitle1"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
```

Comment:
Remove `layout_height` attribute as it is already mentioned in `Body` style. (Make sure its `wrap_content` only in styles.xml)